src/messages.js

```diff
@@ -64,7 +64,11 @@ const messages = {
   changeUrlConfirmation: (oldUrl, newUrl) => `You will now change your node URL from ${oldUrl} to ${newUrl}. Do you wish to continue?`,
   nectarWarning: '⚠️ WARNING! ⚠️ This operation will cost you some nectar (usually less than 0.05 AMB).',
   changeUrlSuccessful: (newUrl) => `Success! Node URL changed to ${newUrl}`,
+  availablePayouts: (availableAmount) => `You can withdraw ${availableAmount} AMB`,
+  confirmWithdraw: 'Would you like to withdraw now?',
+  withdrawalSuccessful: (withdrawnAmount) => `Great! ${withdrawnAmount} AMB (minus small nectar fee) has been transfered to your account.`,
```

Comment:
Maybe add some more sugar here? 😄 Let's do sth like onboarding success message 😛

Reply:
Let's go beyond that. https://www.npmjs.com/package/gif-cli